inductiva/fluids/scenarios/coastal_area/coastal_area.py

```diff
@@ -17,6 +17,8 @@
                                        replace_params_in_template)
 from inductiva.fluids.scenarios.coastal_area.output import CoastalAreaOutput
 
+from inductiva.fluids.scenarios.coastal_area import random_bathymetry_utils
```

Comment:
check Hugo notes on importing functions / classes

Reply:
This follows those notes. Other imports in this file don't, but I'd rather do a general cleanup later.